lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -138,6 +138,7 @@ def add_three_ds(post, payment_method, options)
         post[:consumerAuthenticationInformation][:paSpecificationVersion] = three_d_secure[:version] if three_d_secure[:version]
         post[:consumerAuthenticationInformation][:directoryServerTransactionID] = three_d_secure[:ds_transaction_id] if three_d_secure[:ds_transaction_id]
         post[:consumerAuthenticationInformation][:eciRaw] = three_d_secure[:eci] if three_d_secure[:eci]
+        post[:consumerAuthenticationInformation][:mcc] = options[:mcc] if options[:mcc]
```

Comment:
hey @jcreiff after our discussion I changed it to be be pulling the mcc from the options object however would it pose any issue or concern if line 128 is still there? im thinking theres no point to send the mcc by itself if none of the rest of the threeds fields are there anyways but wanted to run it by you, thanks!

Reply:
I think that guard clause should stay as is - the 3DS info is the main reason to enter this block of logic, and it doesn't seem like we want to entertain the option of only entering this method to add MCC.

However, this brings up a question about the requested use case that initiated this work, which I will follow up on in a separate thread